edgelet/iotedge/test-files/config/ca-certs-est/certd.toml

```diff
@@ -0,0 +1,24 @@
+# This file is auto-generated by `iotedge config apply`
+# Do not edit it manually; any edits will be lost when the command is run again.
+
+homedir_path = "/var/lib/aziot/certd"
+[cert_issuance.aziot-edged-ca]
+common_name = "iotedged workload ca my-device"
+expiry_days = 120
+method = "est"
+url = "https://example.org/.well-known/est"
+username = "user"
+password = "password"
```

Comment:
nit: check that SDL pipeline does not mark it as a secret. Probably need to update .json file with exceptions